python3/cloudwatch/src/lambda_function.py

```diff
@@ -88,11 +89,19 @@ def _parse_to_json(log):
 def _parse_cloudwatch_log(log, additional_data):
     # type: (dict, dict) -> bool
     _add_timestamp(log)
+
     if LAMBDA_LOG_GROUP in additional_data['logGroup']:
         if _is_valid_log(log):
             _extract_lambda_log_message(log)
         else:
             return False
+
+    if _has_whitelist():
```

Comment:
Whitelist & blacklist should be mutually exclusive IMO, doesn't make sense for both to be allowed.

Reply:
Added a warning when both are defined. I think that should be enough?

Reply:
I'd prefer if it will print an error and exit.